File: 프로그래머스/lv1/12901. 2016년/2016년.py
def solution(a, b):
    months = [0,31,29,31,30,31,30,31,31,30,31,30,31]
    # answer = (sum(months[:a])+b)%7
    # if answer == 1:
    #     return "FRI"
    # elif answer == 2:
    #     return "SAT"
    # elif answer == 3:
    #     return "SUN"
    # elif answer == 4:
    #     return "MON"
    # elif answer == 5:
    #     return "TUE"
    # elif answer == 6:
    #     return "WED"
    # else:
    #     return "THU"
        
    week = ["THU","FRI","SAT","SUN","MON","TUE","WED"]
    return week[(sum(months[:a])+b)%7]

File: 프로그래머스/1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        int answer = 0;
        int[] months = new int[]{0,31,29,31,30,31,30,31,31,30,31,30,31};
        String[] week = new String[]{"THU","FRI","SAT","SUN","MON","TUE","WED"};
        for (int i=1; i<a; i++){
            answer += months[i];
        }
        return week[(answer+b)%7];
    }
}